docs/source/index.rst

```diff
@@ -2,54 +2,101 @@
 Dask
 ====
 
-Dask is a simple task scheduling system that uses directed acyclic graphs (DAGs)
-of tasks to break up large computations into many small ones.
+Dask is a flexible parallel computing library for analytics.  Dask emphasizes
+the following virtues:
 
-**Dask enables parallel computing** through task scheduling and blocked algorithms.
-This allows developers to write complex parallel algorithms and execute them
-in parallel either on a modern multi-core machine or on a distributed cluster.
+*  **Familiar**: Provides parallelized NumPy array and Pandas DataFrame objects
```

Comment:
I've looked again and can't find a good spot for them.  I'm not overly concerned here.  It's hard for bag/imperative to convey much information before they're explained.  They're also included in the next section.